schema/spacewalk/postgres/procs/insert_product_vulnerable_packages.sql

```diff
@@ -0,0 +1,63 @@
+--
+-- Copyright (c) 2023 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation.
+--
+
+CREATE OR REPLACE PROCEDURE
+insert_product_vulnerable_packages(package_name_in varchar,fix_version_in varchar,product_cpe_in varchar,cve_name_in varchar)
+AS
+$$
+DECLARE
+    cve_id_val numeric;
+    product_cpe_id_val numeric;
+    vulnerable_pkg_id_val numeric;
+begin
+
+    IF NOT exists(SELECT cve FROM rhnCve cve WHERE cve.name = cve_name_in) THEN
+        INSERT INTO rhncve(id, name)
+        VALUES (nextval('rhn_cve_id_seq'), cve_name_in);
+    END IF;
+
+    SELECT id INTO cve_id_val FROM rhncve WHERE name = cve_name_in;
+
+    IF NOT exists(SELECT c FROM suseOVALPlatform c WHERE cpe = product_cpe_in) THEN
```

Comment:
I think you forgot forgot to replace this code with the call to `lookup_oval_platform`

Reply:
Done.